src/main/drivers/rangefinder/rangefinder_lidarmt.c

```diff
@@ -53,50 +65,55 @@ typedef struct __attribute__((packed)) {
     int32_t distanceMm; // Negative value for out of range
 } mspSensorRangefinderLidarMtDataMessage_t;
 
-static void mtRangefinderInit(rangefinderDev_t * dev) {
+static void doNothingRF(rangefinderDev_t * dev) {
     UNUSED(dev);
 }
 
-static void mtRangefinderUpdate(rangefinderDev_t * dev) {
-    UNUSED(dev);
+static void setDeviceConf(rangefinderType_e mtRangefinderToUse) {
```

Comment:
I implemented it like that so the device configuration getMTRangefinderDeviceConf is set/called only once, whether from the range finder or the optical flow detects because each is called separately. I know repeating a for loop over four elements is not a big deal, but I think still that way is more efficient

Reply:
There is no speed optimization necessary during init phase, unless you get into hundreds of millisecond range. Simple, clean code is important.
In general, premature or unnecessary optimization only causes problems, with negligible gain (Assuming the choice is between ugly fast and slightly slower clean code)